easybuild/framework/easyblock.py

```diff
@@ -214,6 +214,18 @@ def _init_log(self):
 
         self.log = fancylogger.getLogger(name=self.__class__.__name__, fname=False)
 
+        # deprecate self.log.error in easyblocks, self.log.fatal should be used instead
+        def error(msg, *args, **kwargs):
```

Comment:
Not sure yet whether this is actually worth the trouble...

This will not catch all occurrences of the use of the `error()` log method anyway (see for example https://github.com/hpcugent/easybuild-easyblocks/blob/master/easybuild/easyblocks/generic/intelbase.py#L51).

I'm strongly considering just dropping the custom definition of `error` (and `exception` too, while we're at it) all together, and announcing this breaking change on the mailing list (before it gets merged).

This will only affect easyblocks that have not been contributed back, in the sense that the `log.error` still being used in them will no longer raise an `EasyBuildError`, and hence will (hopefully) crash and burn somewhere else rather than with a clean error message...

I think we can get away with this by including it in EasyBuild v2.0 (since people are likely to revisit their in-house easyblocks anyway because of http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.htm .

Thoughts?